source/language/types.rst

```diff
@@ -180,6 +180,20 @@ types for the purposes of casting.
    // Declare a machine-precision float.
    float my_machine_float = 2.3;
 
+.. _void-type:
+
+Void type
+~~~~~~~~~
+
+Subroutines and externs that do not need to return a value may be declared to
```

Comment:
```suggestion
Subroutines and externs that do not return a value may be declared to
```
(to avoid any potential implication that return types could be variable)

Reply:
Removed in https://github.com/openqasm/openqasm/pull/384/commits/73136b3c45e5e6381e771d93c071e8758761a19e